napalm/junos/utils/junos_views.yml

```diff
@@ -251,7 +251,16 @@ junos_lldp_neighbors_detail_view:
 ###
 ### BGP config
 ###
+junos_policy_config_table:
```

Comment:
Looking at the code you added in the class I found it very confusing:

```python
for result, value in all_policies[p]:
    if value is True:
```

But it makes sense looking at this table. I suggest making the name more obvious, something like `junos_policy_nhs_config_table` and, similarly, the fields something better than `result1` and `result2` which don't make much sense.